.github/workflows/gopackage.yml

```
@@ -0,0 +1,40 @@
+name: Python package
+
+on: [ push ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      max-parallel: 5
+      matrix:
+        go-version: [ '1.18', '1.19', '1.20', '1.21', '1.22' ]
+        changed-dir: [ "qase-api-client" ]
+    name: Project ${{ matrix.changed-dir }} - Go ${{ matrix.go-version }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: dorny/paths-filter@v3
+        id: filter
+        with:
+          list-files: 'shell'
+          filters: |
+            changes:
+              - '${{ matrix.changed-dir }}/**'
+      - name: Set up Go ${{ matrix.go-version }}
+        if: steps.filter.outputs.changes == 'true'
+        uses: actions/setup-go@v5
+        with:
+          go-version: ${{ matrix.go-version }}
+      - name: Install dependencies
```

let's keep blank lines between steps
```suggestion
          go-version: ${{ matrix.go-version }}
          
      - name: Install dependencies
```

---

fixed